<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="clientsTable">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="dataTable">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>641</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startServerButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopServerButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
